pandas/io/html.py

```diff
@@ -922,6 +923,7 @@ def _parse(flavor, io, match, attrs, encoding, displayed_only, **kwargs):
     return ret
 
 
+@deprecate_nonkeyword_arguments(version="1.4")
```

Comment:
```suggestion
@deprecate_nonkeyword_arguments(version="2.0")
```

Following the new versioning policy, this deprecation will be enforced in 2.0 (https://dev.pandas.io/docs/development/policies.html)